ror-demo-cluster/images/es/install-ror-es-using-file.sh

```
@@ -1,23 +1,34 @@
 #!/bin/bash -e
 
 function greater_than_or_equal() {
-  [ "$1" = "$(echo -e "$1\n$2" | sort -V | tail -n 1)" ];
+  # Strip the -pre part (or any suffix starting with -) from both versions
+  version_1=$(echo "$1" | sed 's/-pre.*//')
+  version_2=$(echo "$2" | sed 's/-pre.*//')
+  [ "$version_1" = "$(echo -e "$version_1\n$version_2" | sort -V | tail -n 1)" ];
 }
 
-
 if [[ -z "$ES_VERSION" ]]; then
   echo "No ES_VERSION variable is set"
   exit 1
 fi
 
 echo "Installing ES ROR from file..."
 /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch file:///tmp/ror.zip
+ROR_VERSION=$(unzip -p /tmp/ror.zip plugin-descriptor.properties | grep -oP '^version=\K.*')
```

Let's add a check if the ROR_VERSION is not empty

---

Is it done? IMO it'd be nice to have this check to fail fast in case of hypothetical, future changes when the ROR version will be defined in a different file than `plugin-descriptor.properties`